docs/adrs/ADR-1.md

```diff
@@ -13,29 +12,97 @@ Deepansh
+    | ... new field? ... | ... what all new fields are needed? ... |
+
+  6. A table like the above one needs to be created if we decide on saving question level analysis as well for homework quizzes.
+  7. Should the existing ETL Flow code for assessments be reused or a new code file be created for this?
+
 ## Solution(s)
 > If a solution has been discussed then that should be discussed here. If multiple solutions exist, and their discussion is not feasible/practical on discord, then discuss these potential alternatives and their impact.
 > What alternatives are being considered, their benefits, their costs (team resources, money, time frames), mitigations for any drawbacks. The team will collectively take a decision in the Pull Request itself. The agreed upon decision then should be documented in the PR.
 
+Proposed solutions for the questions above:
+  1. After writing this, I think this should be stored in the same `test_responses` table. As the data will be almost similar to the assessments data. The only difference being the `test_type` being `homework`.
+  2. If we store in the same table, this question is redundant.
+  3. TBD
+  4. According to me, same table. Will keep things simpler to manage and the data also makes sense staying in the existing table.
+  5. TBD
+  6. TBD
+  7. Same flow. The current code skips the quizzes which are of `homework` type. The new change to the code will not skip `homework` type quizzes, will take them into consideration and will transform the data accordingly. And because of this, the name of the flow will have to be changed from `ETL Flow Assessments` to something like `ETL Flow Quizzes`.
+
+
 ## Implementation / Tech Specs
 > This section includes the meaty details of the decision. All the technical details, diagrams, specifications, syntax and semantics should be discussed here.
 > This section can be merged with `Solution` if need be. Images can also be attached as shown below
 
-<!-- ![](logo.jpeg) -->
+This is how the ETL flow's architecture will look like.
+
 <p align="center">
-  <img src="/tempReferenceDiagram.webp" width="400">
+  <img src="/adr1_image1.drawio.svg" width="800">
 </p>
 
+
+
 ## Status
 > A decision may be "proposed" if the project stakeholders haven't agreed with it yet, or "accepted" once it is agreed. If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement. You can use any one of these given badges
 
 <Badge text="PROPOSED"/>
-<Badge text="ACCEPTED"/>
-<Badge text="DEPRECATED" type="error"/>
-<Badge text="SUPERSEDED"type="warning"/>
 
 ## Consequences
 > This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
```

Comment:
Maybe add an extra paragraph here about how long ETL flow takes now, and how long you expect it to take when the Homework sessions are also added, since (I think?) there are more homeworks than tests?

Reply:
How long as in? The execution time?

Reply:
Haan execution time. Any other effects. Effect on lambda cost... whatever you can think of :D